config/hocon/src/main/java/io/helidon/config/hocon/HoconConfigParserBuilder.java

```diff
@@ -56,6 +56,16 @@ public HoconConfigParserBuilder disableResolving() {
         return this;
     }
 
+    /**
+     * Enables HOCON resolving substitutions support. Default is {@code false}.
+     *
+     * @return modified builder instance
+     */
+    public HoconConfigParserBuilder enableResolving() {
```

Comment:
Please use method `resolvingEnabled(boolean enabled)` or similar. Having two methods to toggle one option seem excessive.
Maybe deprecate the existing one (if disableResolving exists and is in a public release)

Reply:
Fixed